compiler_gym/envs/mlir/datasets/matmul.py

```diff
@@ -0,0 +1,294 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import logging
+from pathlib import Path
+from typing import Iterable, Optional
+
+from compiler_gym.datasets import Benchmark, BenchmarkSource, Dataset
+from compiler_gym.datasets.benchmark import BenchmarkWithSource
+
+# from compiler_gym.envs.mlir.mlir_benchmark import ClangInvocation
+from compiler_gym.service.proto import BenchmarkDynamicConfig, Command
+from compiler_gym.util.decorators import memoized_property
+from compiler_gym.util.runfiles_path import runfiles_path, site_data_path
+from compiler_gym.util.shell_format import plural
+
+logger = logging.getLogger(__name__)
+
+# The maximum value for the --seed argument to matmul.
+UINT_MAX = (2 ** 32) - 1
+
+_matmul_BIN = runfiles_path("compiler_gym/third_party/matmul/matmul/bin/matmul")
+_matmul_INCLUDES = runfiles_path(
+    "compiler_gym/third_party/matmul/matmul/include/matmul-2.3.0"
+)
+
+matmul_sizes = [
+    (64, 64, 64)
+    # (128, 128, 128),
+    # (1024, 1024, 1024)
```

Comment:
Is this intentional?

Reply:
I removed them. We have improvements in the configuration side in an upcomming PR.

Reply:
Sounds good :)